.github/PULL_REQUEST_TEMPLATE/new_loader.md

```diff
@@ -7,7 +7,7 @@
 Please include the following information at the top level docstring for the dataset's module mydataset.py:
 
 - [ ] Describe annotations included in the dataset
-- [ ] Indicate the size of the datasets (e.g. number files and duration, hours)
+- [ ] Indicate the size of the datasets (i.e. hours)
```

Comment:
Como en DCASE los archivos suelen ser cortos y las datasets a veces indican el numero de archivos en el nombre (urbansound8k, fsd-50k), igual tiene sentido pedir que se especifique el número de archivos?

Reply:
Whoops haha stayed in Spanish without noticing :D Perhaps we can be as complete as possible?
```
(e.g., number of files and their durations, total dataset duration in hours, etc.)
```
?

Reply:
Yeah, I can add number of files as well. My issue with optional `num files` or `hours` is that at the end of the day you don't know how big the dataset is, since a clip can last 1s or 15 min, so it's difficult to asses if it will be useful for a project (e.g. that needs big datasets). The duration in hours is transparent in that sense. What about 
```
(i.e. hours AND optionally others such as number of files)?
```

Reply:
OK, in that case we could be even more direct:
```
Indicate the total duration of the dataset in hours, and optionally also list the number of individual files
```
?